src/libraries/Microsoft.Extensions.Primitives/src/StringValues.cs

```diff
@@ -219,7 +219,7 @@ static string GetJoinedStringValueFromArray(string[] values)
                         length += value.Length;
                     }
                 }
-#if NETCOREAPP || NETSTANDARD2_1
+
                 // Create the new string
                 return string.Create(length, values, (span, strings) => {
```

Comment:
I don't think you can remove the #ifdef here.

string.Create was not introduced for netstandard2.0 and build this project for netstandard2.0 also.

Reply:
As per @eerhardt 's recommendation in other PR you could perhaps try using ValueStringBuilder and keep the condition here. Something like: https://github.com/dotnet/runtime/commit/f07a1d9564f238b3928113d21bfbc43216bc62df